staging/src/k8s.io/apimachinery/pkg/apis/config/v1alpha1/defaults.go

```diff
@@ -29,9 +29,9 @@ func SetDefaults_ClientConnectionConfiguration(obj *ClientConnectionConfiguratio
 		obj.ContentType = "application/vnd.kubernetes.protobuf"
 	}
 	if obj.QPS == 0.0 {
-		obj.QPS = 50.0
+		obj.QPS = 20.0
 	}
 	if obj.Burst == 0 {
-		obj.Burst = 100
+		obj.Burst = 30
```

Comment:
Better not change the default value here.

Reply:
yeah, let's not

Reply:
I think we should keep the same with https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/v1alpha1/defaults.go#L153-L163. Besides,  i have track the commit log,https://github.com/kubernetes/kubernetes/pull/66058/commits/ad1f3e659e8ea9120e65534c43d1f71d68944894,  those value change are not correct
If wrong or we want change the default value, please correct me:blush:

Reply:
Have check again, if using `kube-scheduler default values` seems correct: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/componentconfig/v1alpha1/defaults.go#L257-L265.
So i'm hesitate to it now